MaxText/tests/inference/page_manager_test.py

```diff
@@ -95,7 +99,7 @@ def test_initialization(self):
     self.assertTrue(jnp.all(state.page_status[1:] == 0), "All pages should be initially free (status 0)")
     self.assertTrue(jnp.all(state.num_pages_used[1:] == 0), "No pages should be used initially")
     self.assertTrue(jnp.all(state.sequence_lengths[1:] == 0), "Sequence lengths should be 0 initially")
-    self.assertTrue(jnp.all(state.has_active_page[1:] == False), "No groups should be active initially")
+    self.assertTrue(jnp.all(state.has_active_page[1:] is False), "No groups should be active initially")
```

Comment:
There's a bug here. Before the change, the comparison was elementwise, and returned a boolean array. After the change it checks whether the entire sliced array object is False.

Reply:
Ohh must be an incorrect lint check let me revert